src/components/ForgotPassword/ResetPassword.jsx

```diff
@@ -0,0 +1,152 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { handleForgotPasswordForm, submitResetPassword } from '../../redux/actions';
+import Input from '../common/Input/Input';
+import Button from '../common/Button/Button';
+
+export class ResetPassword extends Component {
+  constructor() {
+    super();
+    this.state = {
+      validPassword: true,
+      passwordError: 'Password must have at least 6 chacters',
+    };
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.onSubmitButton = this.onSubmitButton.bind(this);
+    this.renderErrors = this.renderErrors.bind(this);
+  }
+
+  onSubmitButton(e) {
+    const {
+      onSubmit,
+      newPassword,
+      confirmNewpassword,
+      match: { params },
+    } = this.props;
+    this.setState({ validPassword: true });
+    if (newPassword.length < 5 || confirmNewpassword.length < 5) {
+      this.setState({
+        validPassword: false,
+        passwordError: 'Password must have at least 6 chacters',
+      });
+      return;
+    }
+    if (newPassword !== confirmNewpassword) {
```

Comment:
I thought these checks were all handled in backend?

Reply:
@kimotho-njoki We are handling in the both backend and frontend

Reply:
@kimotho-njoki We are handling in both backend and frontend